server/athenian/api/models/state/models.py

```diff
@@ -100,6 +100,8 @@ class RepositorySet(
     )
     tracking_re = Column(Text(), nullable=False, default=".*", server_default=".*")
     precomputed = Column(Boolean(), nullable=False, default=False, server_default="false")
+    precompute_started = Column(TIMESTAMP(timezone=True))
+    precompute_completed = Column(TIMESTAMP(timezone=True))
```

Comment:
```suggestion
    precompute_finished = Column(TIMESTAMP(timezone=True))
```
Minor syntax nitpick: start-finish; begin-end; initiate-complete. I know that people don't usually care, but this is just good education and conformance to the surrounding table conventions :)

Reply:
Oh, and I see we have the same syntax to fix in `installation_progress` :pray: